_rules/html-page-lang-valid-bf051a.md

```diff
@@ -66,26 +66,18 @@ This `html` element has a `lang` attribute whose value is a [valid][valid langua
 This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
-<html lang="xyz"></html>
+<html lang="enn-US"></html>
```

Comment:
"enn" is a valid language tag. I don't think this example needs to be changed.

Reply:
Changing according to https://github.com/act-rules/act-rules.github.io/issues/1097#issuecomment-573747130
(agreed that "enn" is bad choice).

Reply:
Can someone suggest a better alternative

Reply:
Replaced with `em` which as far as I can tell is not a valid primary language tag.